openmls/src/framing/validation.rs

```diff
@@ -214,7 +214,7 @@ impl DecryptedMessage {
 
 /// Partially checked and potentially decrypted message.
 /// Use this to inspect the [Credential] of the message sender
-/// and the optional `aad` if the original message was an [MlsCiphertext].
+/// and the optional `aad` if the original message was an MlsCiphertext.
```

Comment:
A lot of things became `pub(crate)` recently and therefore it cannot be exposed in the public documentation anymore. That's also the case for `MlsCiphertext`.

Reply:
Right, then referring to MlsCiphertext makes sense in this particular case, because it's a struct from the MLS spec. I don't think that's the case in some of the other places. There, we're referring to structs that are not even visible to the reader. For example in line 264 in this file, we're referring to VerifiableMlsPlaintext. My proposal would be to remove those those references entirely rather than just remove the doc link.